src/main.rs

```diff
@@ -99,6 +97,18 @@ pub struct Args {
     )]
     pub no_hint: bool,
 
+    #[arg(
+        short = 'c',
+        long = "config-file",
+        help = "Path to the configuration file.",
+        long_help = "
+        Specifies the path to the configuration file.
+        ",
+        default_value = "~/.config/jnv/config.toml",
```

Comment:
How about using [directories-rs](https://github.com/dirs-dev/directories-rs) for default?

Reply:
Easy enough